src/ja/common/proxy/remote.py

```diff
@@ -1,9 +1,6 @@
-from abc import ABC
-
-
-class Remote(ABC):
+class Remote(object):
```

Comment:
isn't deriving from object automatic in python?

Reply:
It does.

Reply:
object is indeed the default super-class.
I only added it for clarity.